pkg/api/customization/cluster/actionhandler.go

```diff
@@ -64,6 +66,16 @@ func (a ActionHandler) ClusterActionHandler(actionName string, action *types.Act
 		return a.ImportYamlHandler(actionName, action, apiContext)
 	case "exportYaml":
 		return a.ExportYamlHandler(actionName, action, apiContext)
+	case "viewMonitoring":
+		if !canUpdateCluster() {
```

Comment:
Why do you need to check for update cluster privilege here?

Reply:
I want to ensure accessor has the same privilege as `disableMonitoring` and `editMonitoring`.